docs/explanations/how_to_interpret_the_security_status_command.rst

```diff
@@ -320,6 +320,8 @@ will contain the following attributes:
 * **status**: The status for this update. It will be one of:
 
   * **"upgrade_available"**: The package can be upgraded right now.
+  * **"upgrade_available_not_preferred"**: The package can be upgraded but the
+    upgrade is not the preferred candidate version.
```

Comment:
I'd say we can explicitly say that `the package can be upgraded, but this upgrade will not be automatically selected by APT or other tools - due to lower pinning or to the existence of another preferred update.

Reply:
Updated!